Documentation/configuration.md

```diff
@@ -263,6 +263,12 @@ Follow the instructions when using these flags.
 + Print the version and exit.
 + default: false
 
+## Profiling flags
+
+### -enable-pprof
++ Enable pprof profiler, location is client URL + "/debug/pprof"
```

Comment:
Enable runtime profiling data via HTTP server. Address is at `client URL + "/debug/pprof"`